server/cmake/modules/FindLibClang.cmake

```diff
@@ -16,7 +16,7 @@
 # most recent versions come first
 # http://llvm.org/apt/
 set(LIBCLANG_KNOWN_LLVM_VERSIONS 5.0.0 5.0
```

Comment:
Before this line, could you try to add the following debug information and show me the output for my culture?

```cmake
find_package(Clang)

if (NOT Clang_FOUND)
   message(STATUS "not found: ${CLANG_INCLUDE_DIRS}")
else()
   if (NOT TARGET libclang)
     message(STATUS "libclang not a target???")
   endif()
   get_property(LIBCLANG_LIBRARY1
        TARGET libclang
        PROPERTY LOCATION)
   message(STATUS "Found: ${CLANG_INCLUDE_DIRS} ${LIBCLANG_LIBRARY1}")
endif()
```